src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/mgmt_iot_hub/swagger_iot_hub_resource_provider.json

```diff
@@ -0,0 +1,1913 @@
+{
```

Comment:
Swagger files describing our services normally live in the https://github.com/azure/azure-rest-api-specs repo. Adding it creating a pull request there will ensure that the right people review the swagger file for correctness.